### ISSUE TITLE
Document `rasterize=True` together with `by`

### ISSUE THREAD

**Issue by maximlt** [holoviz/hvplot#1134]
Document `rasterize=True` together with `by`, following the support added in https://github.com/holoviz/hvplot/pull/1132. This was not documented at the time as HoloViews wasn't yet released with the feature.

**Comment by maximlt** [comment]
@Hoxbro @ahuang11 I haven't followed very closely the development of ImageStack in HoloViews and its ramification in Bokeh. Are there enough features released (even in dev versions of HoloViews) to create a nice and useful example with `rasterize`+`by`?

**Comment by philippjfr** [comment]
Yes, you should be able to create nice examples at this point. The only problem now are the hover tooltips.

**Comment by maximlt** [comment]
Ok so let's do that for this release.

**Comment by Hoxbro** [comment]
We still need an official HoloViews 1.18 release before it makes sense.

**Comment by maximlt** [comment]
Ah true, well then it'll have to wait!
